### DISCUSSION TITLE
Unable to run: bcftools

### DISCUSSION THREAD

**Discussion by rengang2020** [biostars/biostar-handbook-issues#283] [General]
Hello Ialbert and everyone! I was following the step by step installation from [https://www.biostarhandbook.com/step-by-step.).

However, when I ran doctor.py, it gave an error for bcftools as shown, can anyone give any suggestion?

$ doctor.py 
# Doctor! Doctor! Give me the news.
# Checking symptoms ...
# bwa           ... OK
# datamash      ... OK
# fastqc -h     ... OK
# hisat2        ... OK
# seqret -h     ... OK
# featureCounts ... OK
# efetch        ... OK
# esearch       ... OK
# samtools      ... OK
# fastq-dump    ... OK
# bowtie2       ... OK
# bcftools      --> ERROR
# seqtk         ... OK
# seqkit        ... OK
# bio           ... OK
# fastq-dump -X 1 -Z SRR1553591 ... OK
#
# Found some problems
#
# Unable to run: bcftools
#
# Check the Biostar Handbook for installation help.

**Comment by rengang2020** [comment]
i solved the problem by running, no worry! 

mamba install gsl=2.5

**Reply by ialbert** [reply]
thanks for following up and providing the solution
